stable/opsportal/values.yaml

```diff
@@ -0,0 +1,53 @@
LTQuMi0zLjMtNi43LTJsLTIuOSwxLjVjLTIuNSwxLjMtMy4yLDMuOC0yLjMsNi42CgkJCQlDMzIzLjUsMjgwLjIsMzQxLjksMzA0LjYsMzQxLjksMzA0LjZMMzQxLjksMzA0LjZ6Ii8+CgkJCTxwYXRoIGlkPSJwYXRoMzI2MCIgaW5rc2NhcGU6Y29ubmVjdG9yLWN1cnZhdHVyZT0iMCIgY2xhc3M9InN0MTIiIGQ9Ik0zMzUsMjczLjdjLTguNS0xNi42LTE0LTMwLjgtMTIuNS0zMi4zbDAtMC4xCgkJCQljMCwwLDAuMSwwLDAuMS0wLjFjMCwwLDAsMCwwLDBsMCwwYzE0LTcuMSwzMi4zLDAuOSw0MSwxNy44YzguNywxNi45LDQuNCwzNi40LTkuNSw0My43bDAsMGMwLDAsMCwwLDAsMGMwLDAtMC4xLDAtMC4xLDAuMQoJCQkJbDAtMC4xQzM1MS45LDMwMywzNDMuNSwyOTAuMywzMzUsMjczLjdMMzM1LDI3My43eiIvPgoJCTwvZz4KCQk8cGF0aCBpZD0icGF0aDMyNjIiIGlua3NjYXBlOmNvbm5lY3Rvci1jdXJ2YXR1cmU9IjAiIGNsYXNzPSJzdDEzIiBkPSJNMzYyLjcsMjY2LjdjMCwwLTEwLjMtMjUuOS0yMi4zLTM5LjUKCQkJYy0xMi0xMy43LTEyOS4xLTE4LjMtMTQ4LjQsMHMtMjYuNSwzOS41LTI2LjUsMzkuNVYyNjFjMCwwLDguOS0yNS4yLDI2LjUtMzkuNXMxMzMuMi0xMy42LDE0OC4xLDBjMTQuOSwxMy42LDIyLjUsMzkuNSwyMi41LDM5LjUKCQkJVjI2Ni43eiIvPgoJCQoJCQk8ZWxsaXBzZSBpZD0iZWxsaXBzZTMyNjQiIHNvZGlwb2RpOmN4PSI0NjIuMDkyMDEiIHNvZGlwb2RpOmN5PSIyNDguMjExIiBzb2RpcG9kaTpyeD0iNi45NDg5OTk5IiBzb2RpcG9kaTpyeT0iMTYuOTc1IiB0cmFuc2Zvcm09Im1hdHJpeCgwLjg4OTggLTAuNDU2MyAwLjQ1NjMgMC44ODk4IC03OC42NjgzIDE5NC4wNzgpIiBjbGFzcz0ic3QxMSIgY3g9IjM2Mi42IiBjeT0iMjU5LjkiIHJ4PSI0IiByeT0iOS45Ij4KCQk8L2VsbGlwc2U+CgkJCgkJCTxlbGxpcHNlIGlkPSJlbGxpcHNlMzI2NiIgc29kaXBvZGk6Y3g9IjEyNS45NjIiIHNvZGlwb2RpOmN5PSIyNDcuMDI5MDEiIHNvZGlwb2RpOnJ4PSI2Ljk0ODk5OTkiIHNvZGlwb2RpOnJ5PSIxNi45NzUiIHRyYW5zZm9ybT0ibWF0cml4KDAuNDgxOCAtMC44NzYzIDAuODc2MyAwLjQ4MTggLTE0MC42Njc5IDI4MC42NjczKSIgY2xhc3M9InN0MTEiIGN4PSIxNjciIGN5PSIyNTkuMyIgcng9IjkuOSIgcnk9IjQiPgoJCTwvZWxsaXBzZT4KCTwvZz4KCTxnIGlkPSJnMzI2OCIgdHJhbnNmb3JtPSJtYXRyaXgoMC42MzkyNjIwNCwwLDAsMC42MzkyNjIwNCwtMjEuMDM5MTI5LC04NC44NzQ4MjcpIj4KCQk8cGF0aCBpZD0icGF0aDMyNzAiIGlua3NjYXBlOmNvbm5lY3Rvci1jdXJ2YXR1cmU9IjAiIGNsYXNzPSJzdDE0IiBkPSJNMzE4LjYsMjYxLjNoLTIxLjZoLTE3LjJoLTI2LjNoLTE0LjloLTI5LjMKCQkJYy0xOS42LDAtMTkuNiw1Mi4xLTEuMSw1NS4xbDM4LjEtMi40YzQuNiwwLDEzLjgtMTcuOCwxNS41LTE5LjJjMS43LTEuNCw3LjQtMS42LDkuNiwwczcuMywxNy40LDExLjksMTcuNGwzOC4xLDQuMgoJCQlDMzM5LjIsMzA4LjIsMzM0LjcsMjYxLjMsMzE4LjYsMjYxLjNMMzE4LjYsMjYxLjN6Ii8+CgkJPHBhdGggaWQ9InBhdGgzMjcyIiBpbmtzY2FwZTpjb25uZWN0b3ItY3VydmF0dXJlPSIwIiBjbGFzcz0ic3QxNSIgZD0iTTIzOC4zLDI3Mi4xYy0xMi0wLjMtMzAuOS0wLjItNDEuMi0wLjEKCQkJYy0xLjcsNS43LTMuNiwxMi44LTIuNywxOS44YzkuMywwLDI5LjcsMCw0NC4xLDBjOC43LDAsMTYsMS42LDIyLjgsMy40YzAuMi0wLjIsMC4zLTAuNCwwLjQtMC41YzEuNy0xLjQsNy40LTEuNiw5LjYsMAoJCQljMC42LDAuNCwxLjQsMS44LDIuMywzLjZjMy41LDAuNyw2LjksMS4yLDEwLjYsMS4zYzkuOCwwLjEsMzIuOSwwLjEsNDcuOCwwYzEuOS02LjMsMC41LTEzLjEsMC4xLTE5LjVjLTE0LjUsMC0zNy41LTAuMy00OC42LTAuMwoJCQlDMjY3LjksMjc5LjksMjU0LjMsMjcyLjQsMjM4LjMsMjcyLjFMMjM4LjMsMjcyLjF6Ii8+Cgk8L2c+Cgk8cGF0aCBpZD0icGF0aDMyNzQiIGlua3NjYXBlOmNvbm5lY3Rvci1jdXJ2YXR1cmU9IjAiIGQ9Ik0xNDEuOCwxMTAuM2MtMC4zLTcuMiwxMy40LTguMSwxNS0yLjEKCQlDMTU4LjQsMTE0LjIsMTQyLjUsMTE1LjYsMTQxLjgsMTEwLjNDMTQxLjMsMTA2LDE0MS44LDExMC4zLDE0MS44LDExMC4zeiIvPgo8L2c+Cjwvc3ZnPgo="> <p>Traefik Dashboard</p></a> </section> </div></main><footer class="footer">© 2019 Mesosphere, Inc. All Rights Reserved</footer>'
```

Comment:
You could put this string into an actual file and load it as a template variable. I did the same thing for `dex-k8s-authenticator` - https://github.com/mesosphere/charts/blob/dev/staging/dex-k8s-authenticator/templates/configmap_templates.yaml#L13

Reply:
not a bad idea, let me look at that, thanks!